resources/public/api/conf/1.0/application.yaml

```diff
@@ -182,20 +208,30 @@ paths:
                     example: "Unexpected Error"
   /monthly/{isaManagerReferenceNumber}/{taxYear}/{month}/declaration:
     post:
-      summary: Declare that the submission is complete
+      summary: Finalise monthly report
```

Comment:
I think want to keep declaration in the summary title, since we're considering it a legal declaration